UPGRADE-2.8.md

```diff
@@ -332,66 +332,81 @@ DependencyInjection
 WebProfiler
 -----------
 
-The `profiler:import` and `profiler:export` commands have been deprecated and
-will be removed in 3.0.
+ * The `profiler:import` and `profiler:export` commands have been deprecated and
+   will be removed in 3.0.
 
-The web development toolbar has been completely redesigned. This update has
-introduced some changes in the HTML markup of the toolbar items.
+   The web development toolbar has been completely redesigned. This update has
```

Comment:
this should be a different bullet point. It is not the same change the deprecating import and export